lxd/main_waitready.go

```diff
@@ -35,15 +36,32 @@ func (c *cmdWaitready) Command() *cobra.Command {
 func (c *cmdWaitready) Run(cmd *cobra.Command, args []string) error {
 	finger := make(chan error, 1)
 	go func() {
-		for {
+		for i := 0; ; i++ {
+			// Log initial attempts at debug level, but use warn
+			// level after the 10'th attempt (about 5 seconds). Then
+			// after the 30'th attempt (about 15 seconds), log only
+			// only one attempt every 10 attempts (about 5 seconds),
+			// to avoid being too verbose.
+			log := logger.Debugf
```

Comment:
Please don't do that, I pretty commonly grep for logger.Debugf and similar functions, this effectively breaks that.

Reply:
Fixed. The code is a bit more verbose, but the alternative would be to encapsulate the switch statements in a function, and that would end up being pretty similar to the log alias.